components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.management.apis/src/main/java/org/wso2/micro/integrator/management/apis/security/handler/BasicSecurityHandler.java

```diff
@@ -63,7 +60,9 @@ public void setName(String name) {
 
     @Override
     protected Boolean authenticate(String authHeaderToken) {
-
+        if (LOG.isDebugEnabled()) {
```

Comment:
Shall we remove this if check just for the code to be simple.

Reply:
done